test/chemistry/lennard_jones_potential_test.py

```diff
@@ -17,7 +17,7 @@
 @fixture(name="test_args")
 def test_args_fixture():
     e = Quantity(2.94e-21 * units.joule)
-    sigma = Quantity(4.10e-10 * units.angstrom)
+    sigma = Quantity(4.10 * units.angstrom)
```

Comment:
How is it possible, that 10 orders of magnitude didn't change the test results?

Reply:
I'm surprised myself. I'll look into it

Reply:
Perhaps, this is the behaviour of `approx`. With the correct `sigma` value of `4.10 Å`, the potential value is `2.65e-13 J` as expected, and with the wrong value of `4.10e-10 Å`, the potential value calculated is `-5.59e-77 J`. The documentation for `approx` says that it uses either `math.isclose` or `numpy.isclose` under the hood. Although only `numpy` gives `True`
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/347fcd69-3a15-451d-995b-0e0244304973)

Reply:
Use it like this:
`assert result_value == approx(2.65e-13, abs=1e-15)`
For some reason, tolerance is overriden by some default value, unless it is absolute.

Reply:
I also came to that conclusion